inclusive-design/wecount.inclusivedesign.ca
# Add the William and Flora Hewlett Foundation as a funder (resolves #340)

## Description
Adds the William and Flora Hewlett Foundation as a funder.


## Steps to test

1. Go to deploy preveiw
2. Scroll down to footer at bottom of webpage
3. Observe the funder section of footer in bottom right corner

**Expected behavior:** 
William and Flora Hewlett Foundation statement and logo should be present.

## Additional information

N/A

## Related issues


resolves #340
